tests/unit/test_routes/test_languages.py

```diff
@@ -0,0 +1,51 @@
+from app.utils import get_error_code_from_status
+
+
+def test_languages(module_client, module_db):
+    client = module_client
+
+    response = client.get('api/v1/languages')
+    for language in response.json['data']:
+        assert (isinstance(language.get('id'), int))
+        assert (isinstance(language.get('name'), str))
+        assert (language.get('name'))
+    assert (response.json['number_of_pages'] is not None)
+
+    # Test trying to get a page of results that doesn't exist
+    too_far = 99999999
```

Comment:
Let's break this out to its own function

Reply:
Same with this one.